opl/opl.c

```diff
@@ -32,7 +32,8 @@ static opl_driver_t *drivers[] =
 
 static opl_driver_t *driver = NULL;
 
-unsigned int opl_sample_rate = 22050;
+static int num_opl_chips = 1;
```

Comment:
Are multiple `num_opl_chips` variables needed or could a single variable work as well?

Reply:
Shouldn't be too difficult to boil them down to one. I'll just pick one to be the "canonical" one and extern the others.